src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/DatePickerIntegrationTests.cs

```diff
@@ -469,6 +468,11 @@ public async Task CanOpenAndCloseUsingKeyboard()
 		[DataRow("ThaiCalendar")]
 		//[DataRow("UmAlQuraCalendar")]
 
+		// HebrewCalendar fails
+		// AssertFailedException: Expected string to be "11", but "13" differs near "3" (index 1).
+		// Expected string to be "February" with a length of 8, but "January" has a length of 7, differs near "Jan" (index 0).
+		//[DataRow("HebrewCalendar")]
+
```

Comment:
FYI this should have hopefully been fixed by https://github.com/unoplatform/uno/pull/5439